### PR TITLE
[ert3] Move logic out of console

### PR DESCRIPTION
**Pull request by markusdregi**
Resolves: #1191

### CODE REVIEW

File: ert3/engine/_run.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                "b": random.gauss(0, 1),
+                "c": random.gauss(0, 1),
+            }
+        }
+        for _ in range(1000)
+    ]
+
+
+def run(workspace_root, experiment_name):
+    experiment_root = Path(workspace_root) / experiment_name
```
Comment (autt) [inline ert3/engine/_run.py:21]: Looks like the variable experiment_root is no longer in use?

Comment (markusdregi) [reply ert3/engine/_run.py:21]: You are right! I think it is a remnant from before using storage! Good catch 👍

File: ert3/engine/_run.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import ert3
+
+from pathlib import Path
```
Comment (oyvindeide) [inline ert3/engine/_run.py:3]: This is unused.

**Comment by oyvindeide** [comment]
It might not fit because we are working in an existing package, but instead of the `_*` pattern for modules, we could do a `src/_ert3` and `src/ert3` pattern instead.

File: ert3/engine/_run.py (hunk 2 above)
Comment (markusdregi) [reply ert3/engine/_run.py]: ✔️

**Comment by markusdregi** [comment]
> It might not fit because we are working in an existing package, but instead of the `_*` pattern for modules, we could do a `src/_ert3` and `src/ert3` pattern instead.

I don't mind, but I suggest we keep that discussion in an issue by itself... Feel free to create it 👍

**Review by autt — APPROVED** [review]

**Pull request merged**